tests/test_settings.py

```diff
@@ -38,26 +40,50 @@ class DummyPreference(Preference):
         default = "default_value"
         validator = text_is_not_empty
 
+    class DummyPreferenceWithCallableValue(Preference):
+        name = "dummy_preference_with_callable_value"
+        default = math.sqrt
+        desc = """\
+            This is a dummy preference with a method passed as value
+            """
+
```

Comment:
Give me some examples

Reply:
I don't think this is required, because the motivation for this was to test what happens if we have a preference that accepts a callable value while printing the Application preference to screen.

I believe we don't require any more test cases for the above class.